ddocs/medic-client/views/contacts_by_type/map.js

```diff
@@ -1,4 +1,9 @@
 function(doc) {
+  var tombstone = false;
+  if (doc.type === 'tombstone' && doc.tombstone) {
+    tombstone = true;
+    doc = doc.tombstone;
+  }
```

Comment:
This seems like it may have unintended consequences, like, online users will start seeing tombstoned contacts. Why is this needed?

Reply:
This is needed so we purge reports about deleted contacts. This was another reason why I chose querying `contacts_by_depth` initially. 
I don't believe online users will start seeing deleted contacts, in webapp the view is always called with specific keys and tombstones emit a prefixed key ("tombstone-person"). The only place where the view is called without keys is in exports. 
Do you think it would be worth to create a new view in `medic` for this purpose?